include/dca/phys/dca_step/cluster_solver/ctaux/accumulator/tp/tp_equal_time_accumulator.hpp

```diff
@@ -866,25 +866,32 @@ inline double TpEqualTimeAccumulator<parameters_type, MOMS_type>::interpolate_ak
   const static double beta = parameters.get_beta();
   const static double N_div_beta = parameters.get_sp_time_intervals() / beta;
 
-  // make sure that new_tau is positive !!
-  double new_tau = tau + beta;
+  int sign = 1;
+  if (tau < 0) {
+    tau += beta;
```

Comment:
Is the intention really to leave tau possibly negative? From below it seems like the answer is no.
How do we enforce |tau| < beta?

Reply:
all the tau values are in (-beta, beta), due to being the difference of two time samples in [0, beta).